### PR TITLE
Insufficient mem molpro

### PR DESCRIPTION
**Pull request by Laxzal**
Molpro can generate an error asking for more Mwords in multiple different ways. One of the outputs can appear like so:

```console
For minimal I/O algorithm in triples, increase memory by 522.58 Mwords to 1522.63 Mwords.

 For full I/O caching in triples, increase memory by 3136.12 Mwords to 4136.17 Mwords.

 A further 111.29 Mwords of memory are needed for the triples to run. Increase memory to 1111.34 Mwords.

 GLOBAL ERROR fehler on processor   0  
```

Currently, our trsh.py will reverse search this test and pick up on the line

```console
 A further 111.29 Mwords of memory are needed for the triples to run. Increase memory to 1111.34 Mwords.
```

And then break the loop and increase the memory by 111.29Mwords (in this example). However, as it can be seen, there is suggestions for larger Mword increases but they are not parsed. Therefore, I have added into the trsh.py check to also look for the line
```console
 For full I/O caching in triples, increase memory by 3136.12 Mwords to 4136.17 Mwords.
```

### CODE REVIEW

**Review by alongd — COMMENTED** [review]
Looking good, thanks!

Do you think we should consider (sum) both the "For full I/O caching in triples" and the "further xxxxx Mwords of memory are needed for the triples"?

File: arc/job/trsh.py [hunk 1]
```diff
@@ -357,11 +358,17 @@ def determine_ess_status(output_path: str,
                     keywords = ['IGNORE_ERROR in the ORBITAL directive']
                     error = 'Unconverged'
                     break
+
```
Comment (kfir4444) [inline arc/job/trsh.py:361]: can you remove this linebreak?

**Review by kfir4444 — COMMENTED** [review]
Thanks @Laxzal, great job!
how does this add solves adding more memory to the job though?

**Comment by alongd** [comment]
@Laxzal, what is the status of this PR?

**Comment by Laxzal** [comment]
I can't test it until Zeus returns. I could try atlas but I'd rather deal with a server I'm familiar with that was having the issue

**Comment by Laxzal** [comment]
@alongd with Zeus back online I have run an input file that previously had a memory issue and is now completing after 1 troubleshoot

File: arc/job/adapters/molpro.py [hunk 2]
```diff
@@ -320,8 +320,14 @@ def set_input_file_memory(self) -> None:
         Set the input_file_memory attribute.
         """
         # Molpro's memory is per cpu core and in MW (mega word; 1 MW ~= 8 MB; 1 GB = 128 MW)
-        self.input_file_memory = math.ceil(self.job_memory_gb * 128 / self.cpu_cores)
-
+        #self.input_file_memory = math.ceil(self.job_memory_gb * 128 / self.cpu_cores)
```
Comment (kfir4444) [inline arc/job/adapters/molpro.py:323]: Can you drop this comment?

**Review by kfir4444 — COMMENTED** [review]
Thanks @Laxzal,
One minor comment :+1:

**Review by kfir4444 — APPROVED** [review]

**Pull request merged**